### ISSUE TITLE
Fix PR #634

### ISSUE THREAD

**Issue by Moro-Code** [cds-snc/platform-forms-client#638]
There are somethings that are missing from PR #634 that should have been addressed before it was merged.

1. Zero tests. We don't have the ability to write cypress tests for the admin page atm but we can in the very least write unit tests which can mock axios calls and state.

2. Critical accessibility issues such as inputs having no label. You can find an export of an axe scan here. 

```JSON
{
   "url":"https://forms-staging.cdssandbox.xyz/en/id/133/settings",
   "extensionVersion":"4.22.1",
   "axeVersion":"4.3.5",
   "standard":"WCAG 2.1 AAA",
   "testingStartDate":"2022-02-13T16:30:23.582Z",
   "testingEndDate":"2022-02-13T16:30:23.582Z",
   "bestPracticesEnabled":true,
   "issueSummary":{
      "critical":1,
      "moderate":1,
      "minor":0,
      "serious":0,
      "bestPractices":1,
      "needsReview":0
   },
   "igtSummary":[
      
   ],
   "failedRules":[
      {
         "name":"label",
         "count":1,
         "mode":"automated"
      },
      {
         "name":"landmark-unique",
         "count":1,
         "mode":"automated"
      }
   ],
   "needsReview":[
      
   ],
   "allIssues":[
      {
         "id":"label",
         "impact":"critical",
         "tags":[
            "cat.forms",
            "wcag2a",
            "wcag412",
            "wcag131",
            "section508",
            "section508.22.n",
            "ACT"
         ],
         "description":"Ensures every form element has a label",
         "help":"Form elements must have labels",
         "helpUrl":"https://dequeuniversity.com/rules/axe/4.3/label?application=AxeChrome",
         "node":{
            "any":[
               {
                  "id":"implicit-label",
                  "data":null,
                  "relatedNodes":[
                     
                  ],
                  "impact":"critical",
                  "message":"Form element does not have an implicit (wrapped) <label>"
               },
               {
                  "id":"explicit-label",
                  "data":null,
                  "relatedNodes":[
                     
                  ],
                  "impact":"critical",
                  "message":"Form element does not have an explicit <label>"
               },
               {
                  "id":"aria-label",
                  "data":null,
                  "relatedNodes":[
                     
                  ],
                  "impact":"serious",
                  "message":"aria-label attribute does not exist or is empty"
               },
               {
                  "id":"aria-labelledby",
                  "data":null,
                  "relatedNodes":[
                     
                  ],
                  "impact":"serious",
                  "message":"aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty"
               },
               {
                  "id":"non-empty-title",
                  "data":{
                     "messageKey":"noAttr"
                  },
                  "relatedNodes":[
                     
                  ],
                  "impact":"serious",
                  "message":"Element has no title attribute"
               },
               {
                  "id":"non-empty-placeholder",
                  "data":{
                     "messageKey":"noAttr"
                  },
                  "relatedNodes":[
                     
                  ],
                  "impact":"serious",
                  "message":"Element has no placeholder attribute"
               },
               {
                  "id":"presentational-role",
                  "data":null,
                  "relatedNodes":[
                     
                  ],
                  "impact":"minor",
                  "message":"Element's default semantics were not overridden with role=\"none\" or role=\"presentation\""
               }
            ],
            "all":[
               
            ],
            "none":[
               
            ],
            "impact":"critical",
            "html":"<input class=\"gc-input-text\" type=\"text\" name=\"newFormOwnerEmail\" value=\"omar.nasr@cds-snc.ca\">",
            "target":[
               "input"
            ],
            "failureSummary":"Fix any of the following:\n  Form element does not have an implicit (wrapped) <label>\n  Form element does not have an explicit <label>\n  aria-label attribute does not exist or is empty\n  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty\n  Element has no title attribute\n  Element has no placeholder attribute\n  Element's default semantics were not overridden with role=\"none\" or role=\"presentation\""
         },
         "is_manual":false
      },
      {
         "id":"landmark-unique",
         "impact":"moderate",
         "tags":[
            "cat.semantics",
            "best-practice"
         ],
         "help":"Ensures landmarks are unique",
         "description":"Landmarks should have a unique role or role/label/title (i.e. accessible name) combination",
         "helpUrl":"https://dequeuniversity.com/rules/axe/4.3/landmark-unique?application=AxeChrome",
         "node":{
            "any":[
               {
                  "id":"landmark-is-unique",
                  "data":{
                     "role":"navigation",
                     "accessibleText":null
                  },
                  "relatedNodes":[
                     {
                        "html":"<nav class=\"gc-admin\">",
                        "target":[
                           ".gc-admin"
                        ]
                     }
                  ],
                  "impact":"moderate",
                  "message":"The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable"
               }
            ],
            "all":[
               
            ],
            "none":[
               
            ],
            "impact":"moderate",
            "html":"<nav><div id=\"skip-link-container\"><a href=\"#content\" id=\"skip-link\">Skip to main content</a></div></nav>",
            "target":[
               "#__next > nav"
            ],
            "failureSummary":"Fix any of the following:\n  The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable"
         },
         "is_manual":false
      }
   ]
}
```

3. CSS issues: While we are focused on functional UI over style we should be at least be able to align a button and a text input 

![Screen Shot 2022-02-13 at 11.36.18 AM.png](https://images.zenhubusercontent.com/5e4d99e2b8781e7790b7e8cc/c95e1f78-2220-4a0b-af0d-c09362bff66d)
